core/java/com/google/census/Census.java

```diff
@@ -41,7 +41,11 @@ public static CensusContext getCurrent() {
     return CONTEXT_FACTORY.getCurrent();
   }
 
-  /** Creates a {@link CensusContext} from the given on-the-wire encoded representation. */
+  /**
+   * Creates a {@link CensusContext} from the given on-the-wire encoded representation. Should be
+   * the inverse of {@link CensusContext#serialize()}. The serialized representation should be
```

Comment:
supernit: make a line break with <p> to separate summary description from details